### PR TITLE
[release-4.7] Update PAO dependencies

### PR DESCRIPTION
**Pull request by cynepco3hahue**
The latest release-4.7 changes include some test fixes that will make CI more stable.

Signed-off-by: Artyom Lukianov <alukiano@redhat.com>

### CODE REVIEW

File: cnf-tests/docgen/e2e.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "[rfe_id:27363][performance] CPU Management Verification of cpu manager functionality Verify CPU usage by stress PODs [test_id:37860] Non-guaranteed POD can work on any CPU": "Checks that non guaranteed pod can use any CPU",
     "[rfe_id:27363][performance] CPU Management Verification that IRQ load balance can be disabled per POD [test_id:36364] should disable IRQ balance for CPU where POD is running": "Checks that the runtime will disable the IRQ load balancing for CPUs used by the guaranteed pod, when the pod has the specific runtime class and annotation",
     "[rfe_id:27363][performance] CPU Management when pod runs with the CPU load balancing runtime class [test_id:32646] should disable CPU load balancing for CPU's used by the pod": "Checks that the runtime will disable the CPU load balancing for the guaranteed pod with the specific runtime class and annotation",
+    "[rfe_id:27363][performance] CPU Management when reserved CPUs specified should run infra containers on reserved CPUs": "XXXXXXXX",
```
Comment (fromanirh) [inline cnf-tests/docgen/e2e.json:25]: is this correct? (XXXXX)

Comment (cynepco3hahue) [reply cnf-tests/docgen/e2e.json:25]: nope, nice catch

Comment (cynepco3hahue) [reply cnf-tests/docgen/e2e.json]: done

File: cnf-tests/TESTLIST.md [hunk 2]
```diff
@@ -95,35 +95,38 @@ The cnf tests instrument each different feature required by CNF. Following, a de
...
-| [performance] Pre boot tuning adjusted by tuned   stalld daemon is running on the host | Checks that the stalld daemon is running on the host | 
-| [performance]RT Kernel  should have RT kernel enabled | Verifies that RT kernel is enabled when configured in the profile. | 
+| [performance]Topology Manager  should be enabled with the policy specified in profile | Verifies that when specifying a topology policy in the profile, that is used by the topology manager. | 
 | [performance] CPU Management Verification of configuration on the worker node  Verify rcu_nocbs kernel argument on the node | Checks that the node has rcu_nocbs kernel argument applied | 
+| [performance] CPU Management Verification of cpu manager functionality Verify CPU usage by stress PODs  Guaranteed POD should work on isolated cpu | Checks that the guaranteed pod will use the isolated CPU, the test relevant only for cases when reserved and isolated CPUs complementary and include all online CPUs | 
+| [performance] CPU Management when reserved CPUs specified should run infra containers on reserved CPUs | XXXXXXXX |
```
Comment (fromanirh) [inline cnf-tests/TESTLIST.md:116]: still XXXXX?

Comment (cynepco3hahue) [reply cnf-tests/TESTLIST.md]: not anymore(forgot to run generate), please review again

**Review by fromanirh — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**